packages/testing/playwright/pages/NodeDetailsViewPage.ts

```diff
@@ -4,15 +4,20 @@ import { expect } from '@playwright/test';
 import { BasePage } from './BasePage';
 import { NodeParameterHelper } from '../helpers/NodeParameterHelper';
 import { EditFieldsNode } from './nodes/EditFieldsNode';
+import { RunDataPanel } from './parts/RunDataPanel';
 
 export class NodeDetailsViewPage extends BasePage {
 	readonly setupHelper: NodeParameterHelper;
 	readonly editFields: EditFieldsNode;
+	readonly inputPanel: RunDataPanel;
+	readonly outputPanel: RunDataPanel;
 
 	constructor(page: Page) {
 		super(page);
 		this.setupHelper = new NodeParameterHelper(this);
 		this.editFields = new EditFieldsNode(page);
+		this.inputPanel = new RunDataPanel(this.getInputPanel());
```

Comment:
If we don't use the getInputPanel for anything other than setup, should we just inline the selector here? What are your thoughts on that?